### ISSUE TITLE
Failed to export as PDF. undefined

### ISSUE THREAD

**Issue by HoeYeungHo** [typora/typora-issues#5471]
- [x] Searched existing issues to avoid creating duplicates.

- [x] Confirmed that it can be reproduced in built-in themes without customized css.

   If it only exists in 3rd party themes or css, you can still report it, but please attach the theme target or the css file. We may not "fix" it, if it is caused by 3rd party themes or css styles that we cannot support.

- [x] Searched http://support.typora.io/

----

**Describe the bug**
Failed to export to PDF, Typora says "Failed to export as PDF. undefined."

**To Reproduce**Steps to reproduce the behavior:

1. Go to 'File'
2. Click on 'Export'
3. Scroll down to 'PDF'

> I am sure the markdown file is very large. I can export chapter by chapter. The PDF has about 120 pages in total and contains many formulas.
>
> Typora is installed under C: disk and I have tried to run in administrator mode, but the error still exists.
>
> When I try to set the margin of exported PDF to 32, it can be exported normally, but cannot be exported with the default 16.

**Expected behavior**
PDF export should succeed.

**Screenshots / Screencasts**
Now I want to show the log file from %APPDATA%/Typora

```
INFO 2022/10/13 15:14:43  [focus] 1
INFO 2022/10/13 15:14:48  [blur] 1
INFO 2022/10/13 15:14:48  [win] blur
INFO 2022/10/13 15:14:51  [focus] 1
INFO 2022/10/13 15:14:59  load file [C:\Users\hoeye\AppData\Local\Temp\Typora\1665645298094-0.8484149087648609.html] for PDF export
INFO 2022/10/13 15:14:59  [blur] 1
INFO 2022/10/13 15:14:59  [win] blur
INFO 2022/10/13 15:15:01  [focus] 1
INFO 2022/10/13 15:15:02  [RenderProcess 1][Log] export PDF to G:\ChromeDownload\output.pdf
Timeout for loading print view
INFO 2022/10/13 15:15:09  generate print window with id null
INFO 2022/10/13 15:15:11  [blur] 1
INFO 2022/10/13 15:15:11  [win] blur
```

**Sample Markdown File**
I've emailed you the file and this error report.

**Desktop (please complete the following information):**
 - OS: Windows 11 64bit

**Typora Version**
Now is 1.4.8. But I try multiple versions all failed to export as PDF on my computer.

**Comment by DeanYang121** [comment]
`INFO 2022/10/13 17:24:18  load file [C:\Users\admin\AppData\Local\Temp\Typora\1665653058083-0.5483033344513946.html] for PDF export
INFO 2022/10/13 17:24:18  [blur] 1
INFO 2022/10/13 17:24:18  [win] blur
ERROR 2022/10/13 17:24:19  Error: ERR_FAILED (-2) loading 'file:///C:\Users\admin\AppData\Local\Temp\Typora\1665653058083-0.5483033344513946.html'
    at rejectAndCleanup (node:electron/js2c/browser_init:161:7647)
    at EventEmitter.stopLoadingListener (node:electron/js2c/browser_init:161:8022)
    at EventEmitter.emit (node:events:526:28)
    at EventEmitter.emit (node:domain:475:12) {
  errno: -2,
  code: 'ERR_FAILED',
  url: 'file:///C:\\Users\\admin\\AppData\\Local\\Temp\\Typora\\1665653058083-0.5483033344513946.html'
}
INFO 2022/10/13 17:24:19  generate print window with id null
INFO 2022/10/13 17:24:20  [focus] 1
INFO 2022/10/13 17:24:20  [RenderProcess 1][Log] export PDF to D:\agent_server.pdf
INFO 2022/10/13 17:24:24  [blur] 1
INFO 2022/10/13 17:24:24  [win] blur
INFO 2022/10/13 17:24:26  [focus] 1
INFO 2022/10/13 17:24:27  [blur] 1
INFO 2022/10/13 17:24:27  [win] blur`



Desktop (please complete the following information):
OS: Windows 10 64bit

typora version
Now is 1.4.8

when I export to pdf，it’s error

**Comment by abnerlee** [comment]
Could you attach or send us a sample md file that is fail to export?

And can empty md file be exported successfully?

**Comment by HoeYeungHo** [comment]
Yes, I have sent your email([hi@typora.io](mailto:hi@typora.io)) on October 13, 2022 with the subject "Failed to export as PDF. undefined # 5471".

This problem is normal in most cases. I export a document smoothly with more than 200000 characters and there are a little formulas in it. The file that cannot be exported also with 200000 characters, but there are a large number of formulas.

**Comment by abnerlee** [comment]
So it only happens for large md file?

**Comment by HoeYeungHo** [comment]
At present, I only found this problem in the document I sent to you, and is normal in other files.

**Comment by abnerlee** [comment]
fixed in v1.5.x
